components/chain-badge.tsx

```diff
@@ -0,0 +1,19 @@
+import Badge from "components/badge";
+
+import { SupportedChainData } from "interfaces/supported-chain-data";
+
+interface ChainBadgeProps {
+  chain: SupportedChainData;
+}
+
+export default function ChainBadge({ chain } : ChainBadgeProps) {
+  return(
+    <Badge
+      label={chain?.chainShortName}
+      className="small-info"
+      style={{
+        backgroundColor: `${chain?.color}90`
```

Comment:
missing a `-` ?

Reply:
@moshmage the color is an hexadecimal value, `90` is an alpha value that can be used to add transparency in hexadecimal colors